### DISCUSSION TITLE
Scene Time Signature

### DISCUSSION THREAD

**Discussion by josiahmaharaj** [leolabs/ableton-js#117] [Q&A] [answered]
Hi all,

Is there a way to set a scene's time signature?
![Screenshot 2024-05-04 at 11 49 15 PM](https://github.com/leolabs/ableton-js/assets/7831833/5f2d104d-3710-4d27-aa7e-f7259c024ade)

I was reviewing the LIVE API docs and I'm not sure if it's possible.
I can set the name and tempo, but the only places that allow the time signature to be set are in the Song and Clip classes.

The goal here is to auto-populate my Set List with name, tempo, and time signature.
Any help would be appreciated!

**Comment by leolabs** [comment, marked as answer]
Afaik, unfortunately this is not possible with the current Live API :/

**Reply by josiahmaharaj** [reply]
Yep, I figured, I also tried renaming the scene with the time and signature information e.g. "song 1; 120bpm; 3/4"... but apparently that was removed in v10 and the new columns are the way to do it. oh well lol. 
Thanks!
